cmd/test/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@
 ]]
 
 find_package(Catch2 REQUIRED)
+find_package(roaring REQUIRED)
```

Comment:
The previous roaring change was reworked in PR #1147 and shouldn't resurface in this PR.